test/test-agent-metadata.js

```diff
@@ -42,32 +42,13 @@ describe('agent interaction with metadata service', function() {
     trace = require('..');
   });
 
-  afterEach(function() {
-    agent.stop();
-  });
-
-  it('should stop when the project number cannot be acquired', function(done) {
-    nock.disableNetConnect();
-    var scope = nock('http://metadata.google.internal')
-                .get('/computeMetadata/v1/project/project-id')
-                .times(2)
-                .reply(404, 'foo');
-
-    agent = trace.start({logLevel: 0});
```

Comment:
Since I've moved project ID error handling in `trace-writer.js`, I've reinstated this test in `test-trace-writer.js`:

https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/pull/378/commits/b7822a1a820172328f3a4ee9bfdc77aa98405a25#diff-fdad2e1d3cceacb6b3585907721d641f